main.go

```diff
@@ -1,31 +1,76 @@
 package main
 
 import (
+	"database/sql"
+	"fmt"
 	"github.com/gin-gonic/gin"
-	"net/http"
+	_ "github.com/lib/pq"
+	"log"
+)
+
+const (
+	DB_USER     = "twvjbojy"
+	DB_PASSWORD = "H0kJZUiDgpgweAWNWdLRgI6v6-IEyxyr"
+	DB_NAME     = "twvjbojy host=john.db.elephantsql.com"
+	DB_HOST     = "john.db.elephantsql.com"
 )
 
 var router *gin.Engine
+var db *sql.DB
 
-func main() {
-	router = gin.Default()
-	initializeRoutes()
-	router.Run()	// Runs on default port 8080
+type patient struct {
+	ID         int    `json:"id"`
```

Comment:
Can you add the database attribute name too? Do this for each attribute. E.g.:
```suggestion
	ID         int    `json:"id" db:"id"`
```

Reply:
Thanks for this one :)